lib/util/template/index.spec.ts

```diff
@@ -85,6 +85,40 @@ describe('util/template/index', () => {
     expect(output).toBe('foo');
   });
 
+  it('has access to basic environment variables (basicEnvVars)', () => {
+    const userTemplate = 'HOME is {{env.HOME}}';
+    const output = template.compile(userTemplate, {});
+    expect(output).toBe(`HOME is ${process.env.HOME ?? ''}`);
+  });
+
+  it('and has access to environment variables (can be exposed with customEnvVariables)', () => {
+    const input = {
+      env: {
+        SHELL: process.env.SHELL,
```

Comment:
```suggestion
        SHELL: '/bin/bash',
```

Can't this just a plain string?